utils/classify.py

```diff
@@ -42,12 +56,19 @@ def matcher(df, substring, column, category):
     return df
 
 
-def classify_individuals(individuals_df):
+def classify_individuals(individuals_df: pd.DataFrame):
```

Comment:
why are you only checking individuals for fossil fuel whereas you check orgs for both fossil fuel and clean energy?

Reply:
Compiling the list of identifying substrings to check for fossil fuel fuel employers involved manually searching to ensure it didn't result in accidentally matching things that don't fit. I did this for fossil fuels first, then did it for clean energy organizations since organizations have a much bigger footprint than individuals, and neglected to also find identifying substrings for clean energy employers.